### DISCUSSION TITLE
Apple Pencil support for iPads

### DISCUSSION THREAD

**Discussion by Amenti** [AvaloniaUI/Avalonia#15006] [General]
Should there be any support for Apple Pencils on iPads right now? It seems "clicks" with a Apple Pencil trigger Pressed animations on Buttons for instance but don't execute the underlying command.

**Comment by maxkatz6** [comment]
Pencil click should be supported in the same way as any other touch or mouse input. Just with different PointerType in the arguments.

**Reply by Amenti** [reply]
Does that mean it should already work and I should create a bug or it will be implemented in the future as you just described it?

**Reply by maxkatz6** [reply]
It should already work.

**Reply by Amenti** [reply]
Ok thanks, I created a bug with a sample project: #15011
